src/mongo/util/net/listen.cpp

```diff
@@ -51,6 +52,23 @@
 
 #endif
 
+// Make sure our Linux has the timer we need
+#if defined(__linux__)
+    #ifndef CLOCK_MONOTONIC_COARSE
+        #error "Your ancient kernel doesn't support CLOCK_MONOTONIC_COARSE: upgrade to v2.6.31 or newer"
```

Comment:
@erh what kernel versions need to be supported today? I'd like to try to resurrect this PR, but need to know what to support